internal/services/machinelearning/machine_learning_compute_cluster_resource.go

```diff
@@ -99,6 +99,14 @@ func resourceComputeCluster() *pluginsdk.Resource {
 				ForceNew: true,
 			},
 
+			"enable_node_public_ip": {
```

Comment:
we typically put enabled on the end of the property name, so for consistentancy with the rest of the provider: 
```suggestion
			"node_public_ip_enabled": {
```

Reply:
Done.